Wikipedia/Code/MWKLanguageLinkController.m

```diff
@@ -232,6 +232,16 @@ - (void)getPreferredLanguageCodes:(void (^)(NSArray<NSString *> *))completion {
     }];
 }
 
+- (void)getPreferredWikiCodes:(void (^)(NSArray<NSString *> *))completion {
+    NSArray<NSURL *> *preferredSiteURLs = [self preferredSiteURLs];
```

Comment:
I'm curious, is there are reason to go through preferredSiteURLs instead of going through preferredLanguages collecting the languageCode of each?

Reply:
For whatever reason I had thought going through siteURLs saved me a step, but now I don't see how I thought that 😅 I'll change, I think it makes more sense to go your way.